text/0000-roadmap-2018.md

```diff
@@ -0,0 +1,412 @@
+
+This period culminates with cutting the beta epoch release at the beginning of August, with RustConf following shortly after.
+
+**Fun**! Having pushed hard up to this point, for the remainder of the year we will relax our tight focus and allow for more exploration, as well as work on fun features like const generics, GATs, and specialization. Rust Belt Rust will happen during this period.
+
+**Reflection**. We’ll leave the month at the end of the year for retrospectives and 2019 planning, with the hope that we can start 2019 with a roadmap already in place.
+
+# Rationale, drawbacks and alternatives
+
+Here’s a high level rationale for the two major themes this year: shipping and community.
+
+## Shipping
+
+The data and discussion quickly coalesced around one of the basic contours for this year: a sense that a lot of important improvements are in the pipeline, and **what’s needed this year is a strong focus on** ***shipping***. As @nikomatsakis put it:
+
+
+> [We did good work in 2017: now we have to make sure the world knows it and can use it.](http://smallcultfollowing.com/babysteps/blog/2018/01/09/rust2018/) (@nikomatsakis)
+
+Or, in @nrc’s more provocative phrasing:
+
+
+> [I want 2018 to be boring.](https://www.ncameron.org/blog/rust-2018/) (@nrc)
+
+This sentiment is the foundation for this year’s technical work, and there’s a striking similarity to the spirit leading up to Rust 1.0. While shipping what’s already in our pipeline may be “boring”, it’s not easy. One of the hardest challenges for us as a community is the same one we faced with 1.0, one that @skade voiced succinctly:
+
+
+> [I’d like us to stabilise imperfect things.](http://yakshav.es/rust-2018/) (@skade)
+
+Shipping requires the discipline to say “no” to exciting new ideas, and to say “yes” to delivering on ones we fear are flawed. We have to iterate toward perfection, providing useful snapshots along the way.
+
+## Community
+
+Our work in 2017, especially during [the impl period](https://blog.rust-lang.org/2017/09/18/impl-future-for-rust.html), was the clear result of a community-wide effort. To do the hard work of “[polishing new things into great things](https://www.ncameron.org/blog/rust-2018/),“ and to keep up with Rust’s continuing growth, we need to double down on our investment in our community. That’s a theme with several facets, including project leadership:
+
+
+> [Rust is a staggeringly large project with a huge and talented community, and we need its leadership structure to reflect that if we are to reach our full potential.](http://aturon.github.io/blog/2018/01/09/rust-2018/) (@aturon)
+
+
+> [I want the community leadership to feel like a well-functioning machine, and that the larger community feels well-represented and can trust the leadership teams.](https://www.ncameron.org/blog/rust-2018/) (@nrc)
+
+continued mentoring:
+
+
+> [I’d like to see a stronger focus [on] mentoring.](https://manishearth.github.io/blog/2018/01/10/rust-in-2018/) (@manishearth)
```

Comment:
note: the `[on]` is no longer necessary, i fixed the typo in my post and it now just says "on"